pages/linux/logname.md

```diff
@@ -0,0 +1,7 @@
+# logname 
+
+> Shows the users login name.
+
+- Display login user name:
```

Comment:
Perhaps `Display the currently logged in user's name` would read better?

Reply:
Yup. Done that.